src/components/PlayerDash.jsx

```diff
@@ -62,6 +73,41 @@ export const PlayerDash = React.createClass({
       });
 
     });
-  }
 
+    this.firebase.getEloDateForCurrentMonth(this.props.params.playerId, 'value',(rawItems) => {
+      var items = [];
+
+      rawItems.forEach( (rawItem) => {
+        var item = rawItem.val();
+        item.id = rawItem.key();
+        items.push(item);
+      });
+
+      var getDataPoint = function getDataPoint(id, item){
```

Comment:
ha. yeah I was happy with that code reduction :wink: `Array.prototype.map()` is awesome for writing succinct js, also `.reduce()`, `.filter()`, `.some()` & `.every()` are related functions that are good to have in your toolbox. And the ternary operator `test == true ? doThis : elseDoThat;` is great for reducing basic `if/else`...